src/Backups/BackupsWorker.cpp

```diff
@@ -392,6 +398,23 @@ struct BackupsWorker::BackupStarter
         if (process_list_element)
             process_list_element_holder = process_list_element->getProcessListEntry();
 
+        // If user has customized backup bandwidth with S3 checksum enabled,
+        // warn for the effective bandwidth mismatch with user's setup
+        if (!query_context->getSettingsRef()[Setting::s3_disable_checksum]
+            && backup_info.backup_engine_name == "S3"
+            && query_context->getBackupsThrottler())
+        {
+            UInt64 queryMaxSpeed = query_context->getBackupsThrottler()->getMaxSpeed();
+            // Note: With S3 checksum enabled, each file is read twice — once for checksum, once for upload.
+            // This effectively halves the usable bandwidth relative to max_backup_bandwidth.
+            LOG_INFO(
```

Comment:
```suggestion
            LOG_WARNING(
```

Reply:
you can get rid of that logging with this:
https://pastila.clickhouse.com/?009edaca/68d9bbbf12f020f553f82d756566dbac#Vw0otGKzXU13cA3DWHNnxA==